master/templates/image_table.html

```diff
@@ -18,7 +18,7 @@
         {% for image in image_table %}
             {% if image.classification == 0 %}
                 <tr>
-                    <td> {{ loop.index  }}          </td>
+                    <td id='image_table_row'> {{ loop.index  }}          </td>
```

Comment:
What does this do?

Reply:
I left you a message about this on Slack. I'll copy the message here:

"... The second Branch contains 3 .js files which I am using - along with the Web browser debugger - to try to get a handle on connecting the image table rows to the image carousel. I have been able to access the image table rows, but the only thing I have been successfully able to do is to hide the image carousel and not access it otherwise. I have placed a comment in profile.html which indicates where the .js files can be tested along with the debugger. The debugger can be opened by pressing "F12" when you have the Web page running and active, and setting the debugger to "console." I can show you what I've done in this regard in class on Tuesday, and will continue working on it until then."

connect29.js prints a message to the debugger console from the image table row when you click on it. That's the purpose of the code in the image table row you asked about. connect30.js shows my attempt to send a message to the console from the carousel which fails. connect31.js shows that I can access the image carousel: I can hide it.

Since I wasn't able to accomplish my goal of connecting the image table to the carousel, instead of submitting nothing I submitted what I had done.

If you need any further information, let me know,.